tailwind.config.mjs

```
@@ -16,5 +13,5 @@ module.exports = {
       },
     },
   },
-  plugins: [],
+  plugins: [require("@catppuccin/tailwindcss")({ defaultFlavour: "mocha" })],
```

ESLint

[error] Require statement not part of import statement.

(@typescript-eslint/no-var-requires)